cmd/collectors/zapi/plugins/quota/quota.go

```diff
@@ -129,6 +129,10 @@ func (my *Quota) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
 
 	if my.client.IsClustered() && my.batchSize != "" {
 		request.NewChildS("max-records", my.batchSize)
+		// Fetching only tree quotas
```

Comment:
does 7-mode need a similar change?

Reply:
I don't think 7mode zapi would support this kind of filtering, only path and volume is possible input as per the doc.

<img width="948" alt="image" src="https://user-images.githubusercontent.com/83282894/157050920-afea5afe-e39b-41c9-80c5-ddeb7394dea9.png">